mls.bib

```diff
@@ -82,3 +82,10 @@ @Manual{Pouzet2006LucidSynchrone30
   year = {2006},
   url = {http://www.di.ens.fr/~pouzet/lucid-synchrone/},
 }
+
+@Manual{GettextManual,
+  author = {Drepper, Ulrich and Meyering, Jim and Pinard, François and Haible, Bruno},
+  title = {GNU gettext tools, version 0.21},
+  year = {2020},
+  url = {https://www.gnu.org/software/gettext/manual/gettext.pdf},
```

Comment:
I'm a bit concerned about how we reference the manual; the title and year is for a specific version, but the URL looks like one that will be continuously updated to the most recent version.

Also, why link all the way down to the PDF, instead of the directory above, giving the reader the freedom to choose a documentation format?

Based on the above, I think this might be a better reference:
```suggestion
  title = {GNU gettext tools},
  url = {https://www.gnu.org/software/gettext/manual/},
```

Of course, potential drawback of such a link is that we don't tie the Modelica specification to a specific version of gettext.  Although a bit risky, I see more advantages than disadvantages in this case.

Reply:
Resolved at latest meeting.